diesel_codegen/Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/diesel-rs/diesel/tree/master/diesel_codegen"
 keywords = ["orm", "database", "postgres", "sql", "codegen"]
 
 [dependencies]
-diesel_codegen_syntex = { version = "0.7.0", default-features = false }
+diesel_codegen_syntex = { path = "../diesel_codegen_syntex", version = "0.7.3", default-features = false }
```

Comment:
If this is a breaking change, the next version should be 0.8.0.

> Am 11.09.2016 um 11:56 schrieb Sergio Benitez <notifications@github.com>:
> 
> In diesel_codegen/Cargo.toml:
> 
> > @@ -10,7 +10,7 @@ repository = "https://github.com/diesel-rs/diesel/tree/master/diesel_codegen"
> >  keywords = ["orm", "database", "postgres", "sql", "codegen"]
> >
> >  [dependencies]
> > -diesel_codegen_syntex = { version = "0.7.0", default-features = false }
> > +diesel_codegen_syntex = { path = "../diesel_codegen_syntex", version = "0.7.3", default-features = false }
> 
> I changed the version numbers to 0.7.3 given that this is a breaking change.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>